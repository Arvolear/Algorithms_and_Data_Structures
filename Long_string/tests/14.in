91195839490939286058557836152582
234095608