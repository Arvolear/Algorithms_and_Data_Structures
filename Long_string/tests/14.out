91195839490939286058558070248190
91195839490939286058557602056974
21348545492701842660964020257303064059856
389566640186343376670944